coding-concepts.js

```diff
@@ -8,39 +8,39 @@
 // --------------------1) What will this log?
 
 var cohort = "Delta 2022"
-// console.log(cohort.split(""))
+console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: .split() should return a string that has placed a space between each letter of the string
+// b) Verify and explain: Using .split returned an array of strings containing the letters of the string. All the letters were separated into their own strings.
 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This will return Hello, LEARN Student! because using string interpolation along with ${} the string "LEARN Student" was inserted into the string "Hello, "
+// b) Verify and explain: What i got back was undefined. The function was declared with a parameter, but (name) was never assigned a value.
```

Comment:
name is passed into the function from the argument on line 22.  Whenever I see undefined and I have a function, I always check if there is a return.